### PR TITLE
Scuemata: Add grafana-cli cue schema validation to CI

### PR DESCRIPTION
**Pull request by dsotirakis**
**What this PR does / why we need it**:

This PR is responsible for adding the `grafana-cli` command to validate scuemata to the CI so it can validate the schemas in every PR or push to `main`.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -95,6 +95,15 @@ steps:
   - initialize
   - lint-backend
 
+- name: validate-scuemata
+  image: grafana/build-container:1.4.1
+  commands:
+  - go run build.go build-cli
```
Comment (aknuds1) [inline .drone.yml:101]: Isn't grafana-cli already built by the build-backend step?

File: scripts/main.star [hunk 2]
```diff
@@ -33,7 +33,8 @@ load(
     'publish_packages_step',
     'notify_pipeline',
     'integration_test_services',
-    'upload_cdn'
+    'upload_cdn',
+    'validate_scuemata'
```
Comment (aknuds1) [inline scripts/main.star:37]: Are you forgetting to include the step in the pipeline?

File: scripts/release.star [hunk 3]
```diff
@@ -32,7 +32,8 @@ load(
     'notify_pipeline',
     'integration_test_services',
     'publish_packages_step',
-    'upload_cdn'
+    'upload_cdn',
+    'validate_scuemata'
```
Comment (aknuds1) [inline scripts/release.star:36]: Are you forgetting to include the step in the pipeline?

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (dsotirakis) [reply .drone.yml:101]: Yeah, didn't see that - changing.

File: scripts/main.star (hunk 2 above)
Comment (dsotirakis) [reply scripts/main.star:37]: Yes

File: scripts/release.star (hunk 3 above)
Comment (dsotirakis) [reply scripts/release.star:36]: Aaaaand yes

**Review by aknuds1 — APPROVED** [review]
LGTM!

**Review by sdboyer — APPROVED** [review]
LGTM

**Pull request merged**